core/src/test/java/org/mapfish/print/attribute/ScalebarAttributeTest.java

```diff
@@ -90,7 +90,7 @@ public void testAttributesFromJson() throws Exception {
         assertEquals("m", value.unit);
         assertTrue(value.geodetic);
         assertTrue(value.lockUnits);
-        assertEquals("Arial", value.font);
+        assertEquals("Liberation Sans", value.font);
```

Comment:
There are some docs about the tests here: https://github.com/mapfish/mapfish-print/tree/master/examples

But we could also add a section in the main readme.